src/scripts/plugin.js

```diff
@@ -158,8 +158,13 @@ const loadPlugin = () => {
       CKEDITOR.dialog.add('binderDialog', dialogConfig);
     },
   });
-
-  CKEDITOR.config.extraPlugins = 'enableBinder';
+  
+  //initialize if not already
+  if(!CKEDITOR.config.extraPlugins)
+    CKEDITOR.config.extraPlugins = '';
+  
+  CKEDITOR.config.extraPlugins += 'enableBinder';
```

Comment:
I think this should be

```js
CKEDITOR.config.extraPlugins += ',enableBinder';
```

Since it uses commas to separate multiple plugins.

Reply:
I do not know how it would handle if there is a leading comma, but it should be better than not having them. Also, if relevant we can pass in a String[] instead but then we would need to initialize extraPlugins to an empty array elsewhere.

Reply:
I added an extra check if it is empty. Add a comma only when it is not.